include/miopen/miopen.h

```diff
@@ -4338,6 +4397,143 @@ MIOPEN_EXPORT miopenStatus_t miopenDropoutBackward(miopenHandle_t handle,
 /** @} */
 // CLOSEOUT DROPOUT DOXYGEN GROUP
 
+// TensorReduce APIs
+/** @addtogroup TensorReduce
+ *
+ *  @{
+ */
+
+/*! @brief Creates the ReduceTensor descriptor object
+ *
+ * @param reduceTensorDesc Pointer to a ReduceTensor descriptor type
+ * @return            miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t
+miopenCreateReduceTensorDescriptor(miopenReduceTensorDescriptor_t* reduceTensorDesc);
```

Comment:
The descriptor should be called `miopenReduceDescriptor_t`since its a descriptor for reduce and not tensors.

Reply:
Yes.  miopenReduceTensorDescriptor_t is a complete simulation to cudnnReduceTensorDescriptor_t.  I also prefer to use a simpler name like miopenReduceDescriptor_t